docs/api_reference.rst

```diff
@@ -0,0 +1,12 @@
+api_reference
+===============
+
+.. automodule:: fileinfo
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+.. automodule:: Surfactant
```

Comment:
```suggestion
.. automodule:: surfactant
```

The `s` needs to be lower case -- I think you are testing on Windows, which is less picky about capitalization matching than the ReadTheDocs build machines (which use Linux).